client/components/gravatar/index.jsx

```diff
@@ -52,10 +62,14 @@ module.exports = React.createClass( {
 		}
 
 		const alt = this.props.alt || this.props.user.display_name;
-		const avatarURL = this._getResizedImageURL( safeImageURL( this.props.user.avatar_URL ) );
+		let avatarURL = this._getResizedImageURL( safeImageURL( this.props.user.avatar_URL ) );
+
+		if ( this.state.failedToLoad ) {
```

Comment:
maybe use a ternary instead to avoid re-running `_getResizedImageURL( safeImgeURL())` unnecessarily?